scripts/strip_record_json_of_pcienumeration.py

```diff
@@ -0,0 +1,95 @@
+# CHIPSEC: Platform Security Assessment Framework
+# Copyright (c) 2023, Intel Corporation
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; Version 2.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Contact information:
+# chipsec@intel.com
+#
+
+# Usage: python strip_record_json_of_pcienumeration.py recording.json
+#     Where recording.json is generated from using "--helper recordhelper" on 
+#     chipsec_main.py or chipsec_util.py
+
+import sys
+from json import dumps, loads
+class Split_Enumeration:
+    def __init__(self, filename:str = "") -> None:
+        self.data = {}
+        self.enum_data = {}
+        self.set_filename(filename)
+        if self.filename:
+            self.load_data()
+        
+    def set_filename(self, filename: str) -> None:
+        self.filename = filename.rstrip(".json")
+        
+    def load_data(self) -> None:
+        if not self.filename:

```

Comment:
How do you feel about this check being inside set_filename instead?

Reply:
I wanted to leave it for the load_data in case you use these functions in a separate function and you want to load the filename after the initialization of the class. In that case I only want to bail if you try to load_data without a file. I don't care before that.